fenics_ice/solver.py

```diff
@@ -776,7 +776,7 @@ def timestep(self, adjoint_flag=1, qoi_func=None ):
     #     Q_vaf.assign(self.Q_vaf)
     #     return Q_vaf
 
-    def forward(self, f):
+    def forward(self, f, verbose=False):
```

Comment:
`def forward(self, f, *, verbose=False):`

Reply:
Does the `*` changes the way we pass the arguments in [run_stable_inference_inv.py#L134](https://github.com/bearecinos/smith_glacier/blob/stable_inference/scripts/run_experiments/run_stable_inference/run_stable_inference_inv.py#L134)?:

J_test = slvr_test.forward([mdl_trn.alpha, mdl_trn.beta], verbose=True)

I did my runs with the function not having the `*` should I run them again?